store/RedisStore.js

```diff
@@ -21,12 +27,20 @@ const lua = `
     redis.call('PEXPIRE', key, timeWindow)
     ttl = timeWindow
   end
+  
+  -- If the key is new or if its incremented value has exceeded the max value and exponential backoff is enabled then set its TTL
+  if ttl == -1 or (exponentialBackoff and current > max) then
+    local backoffExponent = current - max - 1
+    ttl = math.min(timeWindow * math.pow(2, backoffExponent), MAX_SAFE_INTEGER)
```

Comment:
Same here, math.pow is deprecated in Lua I believe:

```suggestion
    ttl = math.min(timeWindow * (2 ^ backoffExponent), MAX_SAFE_INTEGER)
```